external/model-preparation-algorithm/mpa_tasks/extensions/datasets/mpa_cls_dataset.py

```diff
@@ -118,13 +117,13 @@ def evaluate(self,
         if self.class_acc:
             results = np.vstack(results)
             gt_labels = self.get_gt_labels()
-            accuracies = self.class_accuracy(results, gt_labels)
+            accuracies = self.__class_accuracy(results, gt_labels)
```

Comment:
Is there a reason for modifying the name of this?

Reply:
I think this pre-defined class_accuracy should be run only with multi-class dataset. 
1) class_accuracy method will require some modifications to use with multi-label task (it can occur errors)
2) the metric of multi-label/hierarchical classification usually uses mAP, OF1, CF1. So, I don't think multi-label/hierarchical task needs class_accuracy. Rather class_f1_score metric might be more helpful in multi-label/hierarchical classification.